streams/src/main/java/org/apache/kafka/streams/kstream/Windows.java

```diff
@@ -83,19 +83,24 @@ protected Windows segments(int segments) {
         return this;
     }
 
+    // TODO?
```

Comment:
We should remove the emit calls in all the windowing specs for 0.10.0.0 as it is not implemented yet. @mjsax Do you want to do that in this PR, or me / @miguno can do that in a separate PR.